docs/upgrading.asciidoc

```diff
@@ -37,3 +42,75 @@ The table below is a simplified description of this policy.
 |2.0.x |2020-05-14 |2.1.0
 |1.0.x |2020-02-23 |3.0.0
 |====
+
+[[upgrade-to-v5]]
+=== Upgrade to version 5.x
+
+Upgrading from version `4.x` to `5.x` of the RUM Agent introduces some breaking changes.
+This guide outlines the changes and the steps required to upgrade.
+
+[float]
+[[v5-breaking-changes]]
+=== 5.x Breaking changes
+
+Agent name changed to `rum-js`::
+The Agent name has been changed to `rum-js`.
+Because of this change, you may need to upgrade your APM Server version,
+and therefor your Elasticsearch and Kibana version.
+
+Official W3C tracecontext support::
+The RUM Agent supports the official W3C tracecontext `traceparent` header,
+instead of the previously used `elastic-apm-traceparent` header.
+If you're using Elastic backend agents,
+you must upgrade them to a version that also supports the official W3C tracecontext headers.
+
+// https://github.com/elastic/apm-agent-rum-js/pull/664
+// WIP! I still need to clean this up
+Consolidated configurations::
+A single queue is now used to process all events (transactions, errors, etc.).
+This change consolidates two configuration options:
++
+* `throttleInterval` - details to come
```

Comment:
Did you simplify this one?
Its 4 config in total to be precise - errorThrottleLimit, errorThrottleInterval, transactionThrottleLimit, transactionThrottleInterval.

Reply:
Oh, thanks! I missed the other two

Reply:
Also we decided to rename `eventsPerMin` to `eventsLimit`.

Reply:
Awesome, thanks again for the updates 🕺

Reply:
Do we have documentation for `eventsLimit` yet?

Reply:
Added documentation for `eventsLimit` to this PR